backend/functions/cases/cases.function.test.ts

```diff
@@ -39,7 +39,7 @@ describe('Mocking CasesController to get error handling', () => {
     expect(httpErrorSpy).toHaveBeenCalledWith(expect.any(UnknownError));
   });
 
-  //TODO rethink how to trigger a CAMS error for this test
+  // TODO rethink how to trigger a CAMS error for this test
```

Comment:
Is this something we need to address for this merge?

Reply:
No.